src/Microsoft.ML.SearchSpace/Readme.md

```diff
@@ -0,0 +1,72 @@
+# Model builder search space
+
+## Introduction
+This library contains all search-space related code, including creating, updating and sampling. 
+
+![image](https://user-images.githubusercontent.com/16876986/133147775-4fc4ba54-255e-41cf-9079-dc523706893b.png)
```

Comment:
This image has a type-o in it: `CURD` API

Reply:
### Resolved